script/bench/junit-diff.py

```
@@ -0,0 +1,80 @@
+
+    print("\n== per suite (after − before) ==")
+    suites = sorted(
+        set(before_suites) | set(after_suites),
+        key=lambda s: after_suites.get(s, 0) - before_suites.get(s, 0),
+    )
+    for suite in suites:
+        b = before_suites.get(suite, 0.0)
+        a = after_suites.get(suite, 0.0)
+        print(f"  {a - b:+8.2f}s  {fmt(b)} -> {fmt(a)}  {suite}")
+
+    print("\n== 25 largest per-test movers (after − before) ==")
```

Ruff (0.15.21)

[warning] String contains ambiguous `−` (MINUS SIGN). Did you mean `-` (HYPHEN-MINUS)?

(RUF001)